BigBaseV2/src/services/vehicle/persist_car_service.hpp

```diff
@@ -78,3 +80,9 @@ namespace big
 		static big::folder check_vehicle_folder();
 	};
 }
+using namespace std;
+
+namespace JSON
```

Comment:
why a whole namespace for this? remove that and just put that as static member method like the rest